packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageSearchBar.tsx

```diff
@@ -109,31 +108,34 @@ export function TracePageSearchBarFn(props: TracePageSearchBarProps & { forwarde
               <IoLocate />
             </Button>
             <Button
-              className={btnClass}
+              className={cx(btnClass, 'TracePageSearchBar--ButtonUp')}
               disabled={!textFilter}
               htmlType="button"
-              icon={<UpOutlined />}
               data-testid="UpOutlined"
               onClick={prevResult}
-            />
+            >
+              <IoChevronUp />
+            </Button>
             <Button
-              className={btnClass}
+              className={cx(btnClass, 'TracePageSearchBar--ButtonDown')}
               disabled={!textFilter}
               htmlType="button"
-              icon={<DownOutlined />}
               data-testid="DownOutlined"
               onClick={nextResult}
-            />
+            >
+              <IoChevronDown />
```

Comment:
Can we pass this Icon as a prop to the Button component?

Reply:
we can, but it's makes styling icons hard.